src/contents/talks.md

```diff
@@ -1,4 +1,4 @@
-📢 Talks
+🎤 Talks
```

Comment:
I switched to the :microphone: emoji here so that we can use the :megaphone: for the media partners section.